src/main/webapp/scripts/toggle.js

```diff
@@ -73,35 +77,55 @@ function renderKeyword(template, keywordObj) {
   $('#keywords').prepend(keywordHTML);
 }
 
+/**
+ * Loads the current location of the user, and passes the locationObj to the callback function.
+ * Returns a locationObj.
+ */
+function loadLocationObj(callback) {
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(async (position) => {
+      const lat = position.coords.latitude;
+      const lng = position.coords.longitude;
+      locationObj = await loadObject(`${LOCATION_OBJ_URL}?lat=${lat}&lng=${lng}`);
+      callback(locationObj);
+    }, (err) => {
+      console.warn(`ERROR(${err.code}): ${err.message}`);
+      alert("We cannot access your location. Try checking your browswer settings");
+    });
+  }
+}
+
 /**
  * Loads the civic section to the DOM, or alerts the user if there aren't any results for their location.
  */
-async function loadCivicSection() {
-  const lat = getCookie("latitude");
-  const lng = getCookie("longitude");
-  if (lat != "" && lng != "") {
-    try {
-      const civicObj = await loadObject(`${CIVIC_OBJ_URL}?lat=${lat}&lng=${lng}`);
-      const locationObj = buildLocationObj(civicObj);
-      const locationTemplate = await LOCATION_TEMPLATE_PROMISE;
-      renderLocation(locationTemplate, locationObj);
-    } catch (err) {
-      alert("We couldn't find any civic information for your location.");
-    }
+async function loadCivicSection(locationObj) {
+  if (locationObj.Country === "United States") {
+    const address = locationObj2Address(locationObj);
+    const civicObj = await loadObject(`${CIVIC_OBJ_URL}?address=${address}`);
+    const civicLocationObj = buildCivicLocationObj(civicObj);
+    const locationTemplate = await LOCATION_TEMPLATE_PROMISE;
+    renderLocation(locationTemplate, civicLocationObj);
+  } else {
+    alert('Sorry, your current location is not supported');
   }
 }
 
+function locationObj2Address(locationObj) {
```

Comment:
Are we printing the user's address in the website?

Reply:
No, but the civic API now takes an address

Reply:
Also, this would allow the user to input any address in the future